torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -3576,6 +3576,39 @@ Tensor gather_with_keepdimed_indices(const Tensor& input, int64_t dim, const Ten
   return out_fw_grad;
 }
 
+std::tuple<Tensor, Tensor, Tensor> attn_backward(const std::vector<torch::autograd::Variable> &grads,
```

Comment:
@albanD -- is `torch::autograd::Variable` the same thing as `at::Tensor` nowadays?